message_definitions/v1.0/minimal.xml

```diff
@@ -695,6 +698,12 @@
       <entry value="242" name="MAV_COMP_ID_TUNNEL_NODE">
         <description>Component handling TUNNEL messages (e.g. vendor specific GUI of a component).</description>
       </entry>
+      <entry value="243" name="MAV_COMP_ID_ILLUMINATOR">
+        <description>Illuminator #1.</description>
+      </entry>
+      <entry value="244" name="MAV_COMP_ID_ILLUMINATOR_2">
```

Comment:
I still think this is odd to add since MAV_COMP_ID_ILLUMINATOR_2 is not differentiated from MAV_COMP_ID_ILLUMINATOR and we do not need to use it at all

Reply:
I'm also unclear on why we have this

Reply:
Per offline conversation, removed `MAV_COMP_ID_ILLUMINATOR_2`